src/index.css

```diff
@@ -1,13 +1,27 @@
+html {
+  position: relative;
+  min-height: 100%;
+}
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  margin: 0 0 100px;
```

Comment:
If the only margin you want to set is the bottom, you can make this a bit more specific with `margin-bottom: 100px`